features/step_definitions/course_listing_steps.rb

```diff
@@ -0,0 +1,40 @@
+
+Given /^I have a browser on the CSCI home page$/ do
+	@page = HomePage.new @browser
+	@page.goto
+end
+
+When /^I enter csci 2994 into the search field$/ do
+	@page.input_search
+end
+
+And /^I click the search button$/ do
+	@page.click_search
+end
+
+Then /^I can see more than one search results$/ do
+  min_elements = 2
+  @browser.link(:class => "gs-title", 
+    :index => (min_elements-1)).wait_until_present
+end
+
+And /^the first result is titled CSCI-2994 - CSCI Current Topics$/ do
+  @page.should have_link(link, :href => "http://global.cscc.edu/courses/course.asp?Z=&amp;D=B&amp;Q=SP&amp;Y=16&amp;C=CSCI-2994&amp;T=")

```

Comment:
This isn't valid expectation syntax. If you had defined the link on your page object as `bookstore`, then you could have done:

```ruby
expect(@page.bookstore?).to be_true
```